### DISCUSSION TITLE
yticks and xlabels in Stacked bandplot

### DISCUSSION THREAD

**Discussion by WSukmas** [SMTG-UCL/sumo#223] [General]
Dear experts, 

The package works very well. Nice figures. Yet I have several questions:
1) I'd like the yticks of the band structure to be 1, 2, 3, ..., 8 instead of 2.5, 5.0, and so on. How do I customise that?
2) How to insert the Greek letter "Gamma" as the xlabels?
3) Is it possible to plot the total DOS by just its area under curve without its line?

Your response would be greatly appreciated.

[band.pdf](https://github.com/SMTG-UCL/sumo/files/12425651/band.pdf)

**Comment by ajjackson** [comment]
- The tick values are automatically decided by Matplotlib; the algorithm is default behaviour and not directly managed by Sumo. It's also not obvious what a good command-line interface would look like to control such a thing.
- Gamma should be labelled as `\Gamma` in the KPOINTS file
- I don't think it is possible to style lines individually. You can set all linewidths to zero with the style file but that will probably not give the result you're after!
